examples/README.md

```diff
@@ -49,22 +51,22 @@ Route::namespace('Auth')->group(function () {
 });
 ```
 
-### Step 3: Use plugin
+### Step 4: Use plugin
 
 The plugin creates an instance of Laravel accessible globally, and at the same time it registers all the directives.
 
 ```js
-import Vue from 'vue';
-import LaravelPermissions from 'laravel-permissions';
+import Vue from "vue";
```

Comment:
Why double quotes? ESLint like many other standards recommend single quote. Also [Laravel](https://github.com/laravel/laravel) uses single quote in its JavaScript files.

Reply:
Change double quotes to single quotes, little boss 🙄